tox.ini

```diff
@@ -6,3 +6,4 @@ skip_missing_interpreters = true
 passenv = GS_RECORD_MODE GS_CREDS_FILENAME
 deps = -rtest-requirements.txt
 commands = nosetests -vv tests/
+# commands = nose2
```

Comment:
Instead of migrating to `nose2` I'd rather update the test framework and use pytest.

more generaly speaking if this does not introduce anything we should not have it then.